modin/pandas/dataframe.py

```diff
@@ -807,8 +808,8 @@ def _update_var_dicts_in_kwargs(self, expr, kwargs):
             return
         frame = sys._getframe()
         try:
-            f_locals = frame.f_back.f_back.f_locals
-            f_globals = frame.f_back.f_back.f_globals
+            f_locals = frame.f_back.f_back.f_back.f_locals
```

Comment:
Can we add `query` and `eval` back now that we have this fix? I think I was able to get them to work with two `f_back`s.